### ISSUE TITLE
Unable to install Vireo

### ISSUE THREAD

**Issue by brooksbecton** [TexasDigitalLibrary/Vireo#978]
Hello All! 

We were trying to get Vireo installed on a Windows Server. I was following the instructions on the [Install Page ](https://github.com/TexasDigitalLibrary/Vireo/wiki/Install)

I am trying to use the Play! webserver so I ran the `play start` command to kick off the web server. I did run the `play deps --sync`. I can give the output of this also if needed. 

Here are my versions
**OS** - Windows 10
**Play** - 1.3.4
**Postgres** - 10
**Java** - 10.0.1
**Python** - 3.6.5

Here is my `system.out`
```
14:33:05,745 INFO  ~ Starting C:\Users\Administrator\Desktop\Vireo-1.9
:: loading settings :: url = jar:file:/C:/play-1.3.4/framework/lib/ivy-2.4.0.jar!/org/apache/ivy/core/settings/ivysettings.xml
14:33:07,527 INFO  ~ Module spring is available (C:\Users\Administrator\Desktop\Vireo-1.9\modules\spring-1.0.3)
14:33:07,527 INFO  ~ Module less is available (C:\Users\Administrator\Desktop\Vireo-1.9\modules\less-0.9.1)
14:33:12,432 INFO  ~ Precompiling ...
14:33:17,485 ERROR ~ 

@78coc245c
Cannot start in PROD mode with errors

Compilation error (In /app/org/tdl/vireo/model/jpa/HashedBlob.java around line 13)
The file /app/org/tdl/vireo/model/jpa/HashedBlob.java could not be compiled. Error raised is : org.hibernate.engine.SessionImplementor cannot be resolved

play.exceptions.CompilationException: org.hibernate.engine.SessionImplementor cannot be resolved
	at play.classloading.ApplicationCompiler$2.acceptResult(ApplicationCompiler.java:264)
	at org.eclipse.jdt.internal.compiler.Compiler.handleInternalException(Compiler.java:678)
	at org.eclipse.jdt.internal.compiler.Compiler.compile(Compiler.java:522)
	at play.classloading.ApplicationCompiler.compile(ApplicationCompiler.java:300)
	at play.classloading.ApplicationClassloader.getAllClasses(ApplicationClassloader.java:430)
	at play.Play.preCompile(Play.java:605)
	at play.Play.init(Play.java:308)
	at play.server.Server.main(Server.java:160)
```

**Comment by brooksbecton** [comment]
Think I got this fixed. I was on Vireo 1.9 when the latest is 3.0.6. 
I had 1.9 because I followed a link in the install here: https://github.com/TexasDigitalLibrary/Vireo/wiki/Install#1-download that said to go to downloads that only has older versions.

**Comment by brooksbecton** [comment]
Even after installing the latest version of Vireo I am still unable to get the play server running. 

Here is my output in the command line when running `play start`
```C:\Users\Administrator\Desktop\Vireo-3.0.6>play start
~        _            _
~  _ __ | | __ _ _  _| |
~ | '_ \| |/ _' | || |_|
~ |  __/|_|\____|\__ (_)
~ |_|            |__/
~
~ play! 1.3.4, https://www.playframework.com
~ framework ID is %test
~
~ removing pid file C:\Users\Administrator\Desktop\Vireo-3.0.6\server.pid for not running pid 4360
~ using java version "10.0.1"
~ OK, C:\Users\Administrator\Desktop\Vireo-3.0.6 is started
~ output is redirected to C:\Users\Administrator\Desktop\Vireo-3.0.6\logs\system.out
~ pid is 5252
~

C:\Users\Administrator\Desktop\Vireo-3.0.6>WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by org.codehaus.groovy.reflection.CachedClass$3$1 (file:/C:/play-1.3.4/framework/lib/groovy-all-2.3.9.jar) to method java.lang.Object.finalize()
WARNING: Please consider reporting this to the maintainers of org.codehaus.groovy.reflection.CachedClass$3$1
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
Exception in thread "main" play.exceptions.UnexpectedException: Unexpected Error
        at play.Play.start(Play.java:568)
        at play.Play.init(Play.java:309)
        at play.server.Server.main(Server.java:160)
Caused by: javax.persistence.PersistenceException: Unable to build EntityManagerFactory
        at org.hibernate.ejb.Ejb3Configuration.buildEntityManagerFactory(Ejb3Configuration.java:925)
        at org.hibernate.ejb.Ejb3Configuration.buildEntityManagerFactory(Ejb3Configuration.java:900)
        at play.db.jpa.JPAPlugin.onApplicationStart(JPAPlugin.java:204)
        at play.plugins.PluginCollection.onApplicationStart(PluginCollection.java:616)
        at play.Play.start(Play.java:538)
        ... 2 more
Caused by: org.hibernate.cfg.beanvalidation.IntegrationException: Error activating Bean Validation integration
        at org.hibernate.cfg.beanvalidation.BeanValidationIntegrator.integrate(BeanValidationIntegrator.java:154)
        at org.hibernate.internal.SessionFactoryImpl.<init>(SessionFactoryImpl.java:307)
        at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:1799)
        at org.hibernate.ejb.EntityManagerFactoryImpl.<init>(EntityManagerFactoryImpl.java:96)
        at org.hibernate.ejb.Ejb3Configuration.buildEntityManagerFactory(Ejb3Configuration.java:918)
        ... 6 more
Caused by: java.lang.NoClassDefFoundError: javax/xml/bind/JAXBException
        at org.hibernate.validator.engine.ConfigurationImpl.parseValidationXml(ConfigurationImpl.java:252)
        at org.hibernate.validator.engine.ConfigurationImpl.buildValidatorFactory(ConfigurationImpl.java:143)
        at javax.validation.Validation.buildDefaultValidatorFactory(Validation.java:111)
        at org.hibernate.cfg.beanvalidation.TypeSafeActivator.getValidatorFactory(TypeSafeActivator.java:445)
        at org.hibernate.cfg.beanvalidation.TypeSafeActivator.activate(TypeSafeActivator.java:96)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
        at org.hibernate.cfg.beanvalidation.BeanValidationIntegrator.integrate(BeanValidationIntegrator.java:148)
        ... 10 more
Caused by: java.lang.ClassNotFoundException: javax.xml.bind.JAXBException
        at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(Unknown Source)
        at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(Unknown Source)
        at java.base/java.lang.ClassLoader.loadClass(Unknown Source)
        ... 19 more
```

Here is some output from the `logs/system.out` file: 
```
2018-07-02 09:55:46,438 INFO  ~ Starting C:\Users\Administrator\Desktop\Vireo-3.0.6
2018-07-02 09:55:49,514 INFO  ~ Module spring is available (C:\Users\Administrator\Desktop\Vireo-3.0.6\modules\spring-1.0.3)
2018-07-02 09:55:49,514 INFO  ~ Module less is available (C:\Users\Administrator\Desktop\Vireo-3.0.6\modules\less-0.9.1)
2018-07-02 09:56:02,374 INFO  ~ Precompiling ...
2018-07-02 09:57:15,960 ERROR ~ Can't start in PROD mode with errors
javax.persistence.PersistenceException: Unable to build EntityManagerFactory
	at org.hibernate.ejb.Ejb3Configuration.buildEntityManagerFactory(Ejb3Configuration.java:925)
	at org.hibernate.ejb.Ejb3Configuration.buildEntityManagerFactory(Ejb3Configuration.java:900)
	at play.db.jpa.JPAPlugin.onApplicationStart(JPAPlugin.java:204)
	at play.plugins.PluginCollection.onApplicationStart(PluginCollection.java:616)
	at play.Play.start(Play.java:538)
	at play.Play.init(Play.java:309)
	at play.server.Server.main(Server.java:160)
Caused by: org.hibernate.cfg.beanvalidation.IntegrationException: Error activating Bean Validation integration
	at org.hibernate.cfg.beanvalidation.BeanValidationIntegrator.integrate(BeanValidationIntegrator.java:154)
	at org.hibernate.internal.SessionFactoryImpl.<init>(SessionFactoryImpl.java:307)
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:1799)
	at org.hibernate.ejb.EntityManagerFactoryImpl.<init>(EntityManagerFactoryImpl.java:96)
	at org.hibernate.ejb.Ejb3Configuration.buildEntityManagerFactory(Ejb3Configuration.java:918)
	... 6 more
Caused by: java.lang.NoClassDefFoundError: javax/xml/bind/JAXBException
	at org.hibernate.validator.engine.ConfigurationImpl.parseValidationXml(ConfigurationImpl.java:252)
	at org.hibernate.validator.engine.ConfigurationImpl.buildValidatorFactory(ConfigurationImpl.java:143)
	at javax.validation.Validation.buildDefaultValidatorFactory(Validation.java:111)
	at org.hibernate.cfg.beanvalidation.TypeSafeActivator.getValidatorFactory(TypeSafeActivator.java:445)
	at org.hibernate.cfg.beanvalidation.TypeSafeActivator.activate(TypeSafeActivator.java:96)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at org.hibernate.cfg.beanvalidation.BeanValidationIntegrator.integrate(BeanValidationIntegrator.java:148)
	... 10 more
Caused by: java.lang.ClassNotFoundException: javax.xml.bind.JAXBException
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(Unknown Source)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(Unknown Source)
	at java.base/java.lang.ClassLoader.loadClass(Unknown Source)
	... 19 more
```

**Comment by jcreel** [comment]
This could be a problem with your Java version.  This older version of Play! is probably not compatible with Java 10.  I'd try downgrading to a JDK somewhere between 5 and 7.

**Comment by thebetternewt** [comment]
I've picked up this project from @brooksbecton. I've attempted to install Vireo with Java 7. However, I've encountered an issue with Java Play 1. 

The latest Java Play 1 is version 1.5.x. 

Java Play versions 1.4.x & 1.5.x require Java 8.

Due to security issues, the urls for some dependencies for Java Play 1.3.x (which used Java 7), are no longer accessible. (Reference: https://github.com/playframework/play1/issues/1126).

So, I cannot install Vireo 3 without Java 7 or earlier, and I can't install Java Play 1 because the currently supported (and working) version requires Java 8.

That being said, it seems that Vireo 3 is not available to be installed as is. Does anyone have a suggestion for how to proceed? Thanks!
